recipes/ipychart/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "ipychart" %}
+{% set version = "0.1.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 0e188841f87379bf01105fd6b5d4649a8183d6de2be2c8b7677ea2b7d750ce7c
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - numpy ==1.19.1
```

Comment:
Do all of these dependencies need to be pinned so tightly? Generally that can cause issues when trying to solve for an environment, so if a less stringent set is allowable, I'd change it to something like:

`- pandas >=1.1.1,<1.2`

for all of them to at least allow minor version updates to be pulled in.

Reply:
Indeed there is not necessarily need to be so strict on the versions. Thank you for your precious help!

Reply:
Obviously the build does not like the new versions that I put. What am I doing wrong here?

![image](https://user-images.githubusercontent.com/25349465/91669294-1278c780-eb14-11ea-825b-344ba03e8c7c.png)

Reply:
You want `- numpy >=1.18.0,<1.20.0`, I think. `>=1.18.0,<1.2` is unsatisfiable since the upper bound is smaller than the lower bound.

Reply:
I'm getting tired ... , I fixed that in my last commit. Thanks!